website/static/js/contribManager.js

```diff
@@ -462,11 +423,39 @@ var ContributorsViewModel = function(contributors, adminContributors, user, isRe
 
 function ContribManager(selector, contributors, adminContributors, user, isRegistration, table, adminTable) {
     var self = this;
+    //shouter allows communication between ContribManager and ContribRemover, in particular which contributor needs to
+    // be removed is passed to ContribRemover
+    var contribShouter = new ko.subscribable();
```

Comment:
And is that happening because you are getting a JS error? If you get a JS error, the ViewModel probably isn't binding, so the template won't look right.